src/utils/run-build.mjs

```diff
@@ -0,0 +1,129 @@
+// @ts-check
```

Comment:
Moved from `src/commands/dev/dev.mjs` mostly unchanged. It just adds two exported functions for the different timelines in Netlify Build, both of which use the existing `runBuild`.